torch_xla_py/xla_dist.py

```diff
@@ -402,3 +407,241 @@ def get_cluster(self):
     cluster.validate()
     return cluster
 
+
+class DistributedExecutor(object):
+
+  SCRIPT_PATH_TMPL = '/tmp/dist_training_ptxla_{worker}.sh'
```

Comment:
I am always nervous when software creates fixed path temporary files into `/tmp`.
Let's put a PID there, and be sure they get removed.